Apps/Admin/Server/Services/SupportService.cs

```diff
@@ -208,7 +208,7 @@ public async Task<IList<PatientSupportResult>> GetPatientsAsync(PatientQueryType
         public async Task BlockAccessAsync(string hdid, IEnumerable<DataSource> dataSources, string reason, CancellationToken ct = default)
         {
             BlockAccessCommand blockAccessCommand = new(hdid, dataSources, reason);
-            await this.patientRepository.BlockAccess(blockAccessCommand, ct).ConfigureAwait(true);
+            await this.patientRepository.BlockAccessAsync(blockAccessCommand, ct).ConfigureAwait(true);
```

Comment:
would be nice to remove configureawait too

Reply:
Changed